admin/data/sql/migrations/017.do.user-school-trigger.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TRIGGER user_school_id_null_check
```

Comment:
Question: should we include the trigger conditions in the name?  e.g. user_school_id_null_check_insert_update ?